src/test/java/org/influxdb/TestUtils.java

```diff
@@ -8,38 +8,41 @@
 
 public class TestUtils {
 
-	public static String getInfluxIP() {
-		String ip = "127.0.0.1";
-		
-		Map<String, String> getenv = System.getenv();
-		if (getenv.containsKey("INFLUXDB_IP")) {
-			ip = getenv.get("INFLUXDB_IP");
-		}
-		
-		return ip;
+	private static String getEnv(String name, String defaultValue) {
+	  Map<String, String> getenv = System.getenv();
+
+	  if (getenv.containsKey(name)) {
+      return getenv.get(name);
+    } else {
+      return defaultValue;
+    }
+	}
+  
+  public static String getInfluxIP() {
+		return getEnv("INFLUXDB_IP", "127.0.0.1");
 	}
 	
 	public static String getRandomMeasurement() {
 		return "measurement_" + System.nanoTime();
 	}
 	
 	public static String getInfluxPORT(boolean apiPort) {
-		String port = "8086";
-		
-		Map<String, String> getenv = System.getenv();		
 		if(apiPort) {		
-			if (getenv.containsKey("INFLUXDB_PORT_API")) 
-				port = getenv.get("INFLUXDB_PORT_API");
+			return getEnv("INFLUXDB_PORT_API", "8086");
 		}
 		else {
-			port = "8096";
-			if (getenv.containsKey("INFLUXDB_PORT_COLLECTD")) 
-				port = getenv.get("INFLUXDB_PORT_COLLECTD");
+			return getEnv("INFLUXDB_PORT_COLLECTD", "8096");
 		}			
-		
-		return port;
 	}
 
+	public static String getProxyApiUrl() {
+    return getEnv("PROXY_API_URL", "http://127.0.0.1:8086/");
```

Comment:
can you please check formatting, here identation is broken

Reply:
fixed it, we should have check style for test as well